docs/_templates/version.html

```diff
@@ -1 +1 @@
-<span class="nv-version">{{ release }}</span>
+<span class="nv-version">{{ release.split("+")[0]}}</span>
```

Comment:
I don't think we should remove the hash. It indicates that its a development build, and tells you exactly which version of the main branch you are looking at. 

The git hash doesn't appear in the `stable` version of the docs, so I think this change is uncessary